forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -119,17 +124,32 @@ def get():
             current_app.logger.critical(err)
             return response, status
 
+        except KeyError as err:
+            response, status = (
+                {
+                    "type": "Invalid Request Object",
+                    "message": "Required fields are not passed",
+                },
+                HTTPStatus.BAD_REQUEST,
+            )
+            current_app.logger.critical(response)
+            current_app.logger.critical(err)
+            return response, status
+
 
 @cors_preflight("GET,PUT,DELETE,OPTIONS")
 @API.route("/<int:application_id>", methods=["GET", "PUT", "DELETE", "OPTIONS"])
 class ApplicationResourceById(Resource):
-    """Resource for submissions."""
+    """Resource for getting application by id."""
 
     @staticmethod
     @auth.require
     @profiletime
-    def get(application_id):
-        """Get application by id."""
+    def get(application_id: int):
+        """Get application by id.
+
+        : application_id:- List the application for particular application_id
+        """
         try:
             if auth.has_role([REVIEWER_GROUP]):
```

Comment:
We added a role, "manager", for the supervisors. May need to update this if planned to return applications for that role in the future. Don't think have an issue at the moment.

Reply:
I'm not seeing any mention of the 'Manager' role in this file on other branches. I'll look for where to re-add it though.

Reply:
@david-sabadash @iman-jamali-fw We should keep this as the `REVIEWER` group. The manager role does not have any additional permissions